stdlib/universe/mode.go

```diff
@@ -240,44 +242,35 @@ func (t *modeTransformation) doString(cr flux.ColReader, tbl flux.Table, builder
 		}
 	}
 
-	if numNil > max {
+	// if every value occurs the most number of times or all values are null, there is no mode
```

Comment:
Is there a reason for `every value occurs the most number of times ... there is no mode`?

If all values are null, then shouldn't `len(stringMode) == 0`?

Reply:
Definitionally, if every value occurs the same/most number of times, there is no mode. 

len(stringMode) == 0 is equivalent to total == 0 since total is initialized to 0, and if the len(stringMode) is 0, the loop on line 237 never executes, so total stays 0.